### DISCUSSION TITLE
Taskbar widget change

### DISCUSSION THREAD

**Discussion by markox92** [ramensoftware/windhawk-mods#800] [Q&A]
Anyone know how I can style widget in taskbar, I want to have one line text like I have on clock

![Screenshot 2024-07-21 195003](https://github.com/user-attachments/assets/13a71d19-543d-49f0-ab78-96923ab61f40)
![Screenshot 2024-07-21 195014](https://github.com/user-attachments/assets/dce38887-31fb-4243-be42-0001e2b7cfb6)

**Comment by m417z** [comment]
The basic idea is to use UWPSpy and find the right target. For more help, a better place to ask about styling the taskbar is here:
https://github.com/ramensoftware/windows-11-taskbar-styling-guide
